plankton/devices/chopper/interfaces/epics_interface.py

```diff
@@ -21,18 +21,65 @@
 
 
 class ChopperEpicsInterface(EpicsAdapter):
+    """
+    ESS chopper EPICS interface
+
+    Interaction with this interface should happen via ChannelAccess (CA). The PV-names
+    usually carry a prefix which depends on the concrete device and environment, so
+    it is omitted in this description. The dynamically generated description of the PVs
+    does however contain the prefix so that the names can be copy-pasted easily.
+
+    The first step is to initialize the chopper, for example via caput on the command line:
+
+        $ caput CmdS init
+
+    After this, the chopper is in a state where it can be started:
+
+        $ caget State
+        State                          stopped
+
+    To set a specific speed and phase, the setpoints have to be configured via caput:
+
+        $ caput Spd 100
+        $ caput Phs 34.5
+
+    Then the chopper can be commanded to move towards those values:
+
+        $ caput CmdS start
+
+    Now the disc accelerates to the setpoints, the state should now be different:
+
+        $ caget State
+        State                          accelerating
+
+    The possible commands are part of the PV-specific documentation.
+    """
     pvs = {
-        'Spd-RB': PV('target_speed', read_only=True),
-        'Spd': PV('target_speed'),
-        'ActSpd': PV('speed', read_only=True),
+        'Spd-RB': PV('target_speed', read_only=True,
+                     doc='Readback value of the speed setpoint in Hz.'),
+        'Spd': PV('target_speed',
+                  doc='Speed setpoint in Hz.'),
+        'ActSpd': PV('speed', read_only=True,
+                     doc='Current rotation speed of the chopper disc in Hz.'),
+
+        'Phs-RB': PV('target_phase', read_only=True,
+                     doc='Readback value of phase setpoint in degrees.'),
+        'Phs': PV('target_phase',
+                  doc='Phase setpoint in degrees.'),
+        'ActPhs': PV('phase', read_only=True,
+                     doc='Current phase of the chopper disc in degrees.'),
 
-        'Phs-RB': PV('target_phase', read_only=True),
-        'Phs': PV('target_phase'),
-        'ActPhs': PV('phase', read_only=True),
+        'ParkAng-RB': PV('target_parking_position', read_only=True,
+                         doc='Readback value of the discs parking position setpoint in degrees.'),
+        'ParkAng': PV('target_parking_position',
+                      doc='The discs parking position setpoint in degrees.'),
+        'AutoPark': PV('auto_park',
+                       doc='If enabled, the chopper disc will be moved to the parking '
+                           'position automatically when the speed is 0 or the chopper '
+                           'is otherwise stopped. 0 means False, 1 means True, the string '
+                           'representations of the enum values are "false" and "true".',
+                       type='enum', enums=['false', 'true']),
```

Comment:
They are not always properties, but plain data members instantiated after the device is constructed. They can't have a docstring there. I only used `doc=...` where it wasn't otherwise possible.